ethdb/pebble/pebble.go

```diff
@@ -238,7 +238,7 @@ func New(file string, cache int, handles int, namespace string, readonly bool, e
 		fn:           file,
 		log:          logger,
 		quitChan:     make(chan chan error),
-		writeOptions: &pebble.WriteOptions{Sync: !ephemeral},
```

Comment:
I think I'll feel better if our default will stay "on" and we could later think if the logic that goes for geth turning it off also fits us